tracker-db/backup/backup.yaml

```diff
@@ -0,0 +1,284 @@
+---
+kind: Template
```

Comment:
Don't need this backup. The lear db backup can back this db as well.

Reply:
removed